### ISSUE TITLE
Time interval for tracking changes separately

### ISSUE THREAD

**Issue by nmueller18** [fiduswriter/fiduswriter#1238]
Currently, it seems that each 10 min, a new time stamp for tracked changes is generated. This behaviour leads to a proliferation of "tracked changes"-badges (related to this [issue](https://github.com/fiduswriter/fiduswriter/issues/1213)) when one works, e. g., on a paragraph for a longer period of time. Also when you return to an older tracked change and insert text, a new badge is generated.

Is it possbile to prolong the interval between new time stamps, perhaps on the server side? An even better solution would entail the incorporation of further tracked changes in an already existing badge if this has been generated by the same user (this is, for example, the behaviour of Pages).

**Comment by johanneswilm** [comment]
> Is it possbile to prolong the interval between new time stamps, perhaps on the server side?

The creation of these badges happens entirely on the client side, by taking the current timestamp and removing the last few digits. 

> An even better solution would entail the incorporation of further tracked changes in an already existing badge if this has been generated by the same user (this is, for example, the behaviour of Pages).

We could remove the timing part entirely, but that would then mean that all continuous changes within the same paragraph would be seen as just one change. I'm not sure that would be desired by all users. The behavior as it currently is was what several popular word processors used a few years ago. I have not checked back whether that is still the case.

**Comment by nmueller18** [comment]
> The behavior as it currently is was what several popular word processors used a few years ago. 

Yes, I have checked and Pages as well as OpenOffice also save time stamps. Actually, this is a debatable as it discloses working hours to colleagues which could at best lead to teasing comments. I cannot think of a really good reason why the knowledge of the exact hour of a certain tracked change is relevant in academics but I might be wrong, and I agree, it is standard behaviour in word processors. That was my point in asking for a server side solution: Perhaps a switch in the server settings to turn on/off the timing part.

**Comment by johanneswilm** [comment]
The server is generally not aware of how the tracked changes work. It is only able to do some checking that no malicious changes are being submitted. 

But server-wide settings can also be made to work in the frontend. We could make removing timestamps entirely a configuration setting - either on a per server, per user or per document level. Not sure what would be the best. It would be good to hear from someone/anyone who actually relies on having those time stamps so we can define distinct usecases and put the setting in the right place.

**Comment by nmueller18** [comment]
As fun fact, the search "Why are time stamps necessary in tracked changes" yields lots and lots of results how to _remove_ time stamps but not one where it is explained why they are necessary (except legal texts).
From a practical point of view, I would vote for a per server solution. Otherwise, it would be a further option for the user to ponder.
